packages/web/hooks/use-export.ts

```
@@ -263,87 +279,90 @@ export function useExport() {
+            return `<img key={${key}} src="${node.url}" alt="${node.alt || ''}" />`
+          case 'table':
+            return `<table key={${key}}>${text}</table>`
+          case 'tr':
+            return `<tr key={${key}}>${text}</tr>`
+          case 'td':
+            return `<td key={${key}}>${text}</td>`
+          case 'th':
+            return `<th key={${key}}>${text}</th>`
+          default:
+            // Handle text formatting
+            let formattedText = text
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)